src/gromacs/nblib/atoms.cpp

```diff
@@ -48,29 +48,29 @@
 namespace nblib {
 
 Atom::Atom() noexcept :
-  name_(""),
-  mass_(0),
-  charge_(0),
-  c6_(0),
-  c12_(0)
+  name_(AtomKindName("")),
```

Comment:
This should just be AtomName.

Reply:
We should clarify names here. In the `Molecule::addAtom(...)` method, the first argument is the AtomName as its relative role within the molecule. Say "H2" in a water molecule which is actually an atom of kind "Hw". 

`AtomKindName` and `AtomName` make this distinction as @sebkelle1 pointed out previously.